libs/feature/record/src/lib/state/mdview.selectors.ts

```diff
@@ -64,3 +64,11 @@ export const getAddUserFeedbacksLoading = createSelector(
   getMdViewState,
   (state: MetadataViewState) => state.addUserFeedbackLoading
 )
+
+/*
+  Feature Catalog Selectors
+*/
+export const getFeatureCatalog = createSelector(
+  getMdViewState,
+  (state: MetadataViewState) => state.featureCatalog
+)
```

Comment:
This needs a test

Reply:
@Guillaume-d-o maybe you can check my PR https://github.com/geonetwork/geonetwork-ui/pull/1137, I hope we won't do the work twice.